packages/react/test/specs/commonTests/isConformant.tsx

```diff
@@ -53,40 +58,40 @@ export default (Component, options: Conformant = {}) => {
 
   const componentType = typeof Component
 
-  // This is added because the component is mounted
+  const helperComponentNames = [
+    ...[ThemeProvider, FelaTheme, Ref, RefFindNode],
+    ...(usesWrapperSlot && [Box]),
```

Comment:
A bit concern with this... I know currently we use the Box for wrapper, but maybe in the config we can provide which Component is used as Wrapper, having default to be Box.